test/setupTests.js

```diff
@@ -5,7 +5,9 @@ jest.mock('path-exists')
 jest.mock('fs')
 jest.mock('ora', () =>
   jest.fn().mockReturnValue({
-    start: jest.fn(),
+    start: jest.fn(function () {
+      return this
+    }),
```

Comment:
🤔

Reply:
Fix for: `UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'fail' of undefined`

The way spinners are used inside the hooks was by chaining the constructor and start method.
Ora's [start()](https://github.com/sindresorhus/ora#starttext) method returns its instance, allowing to then invoke other methods such as `succeed()` or `fail()`.
But if the mocked implementation of `start()` does not return its instance, then`spinner.fail()` or `spinner.succeed()` fails with the error message above